WORKSPACE

```diff
@@ -194,7 +194,7 @@ load(
 go_rules_dependencies()
 
 go_register_toolchains(
-    go_version = "1.12.8",
+    go_version = "1.13.4",
```

Comment:
This needs to be updated manually?

Reply:
Yes. This tells bazel which golang to use. It does not use the installed golang version. It has its own hermetic packages.